deployment/postgres-init-db/sql/views/merlin/derived_events.sql

```diff
@@ -0,0 +1,60 @@
+create or replace view merlin.derived_events
+as
+select source_key,
+  derivation_group_name,
+  event_key,
+  duration,
+  event_type_name,
+  start_time,
+  source_range,
+  valid_at
```

Comment:
please expose `properties`

Reply:
Addressed in [expose 'properties' field in derived_events view](https://github.com/NASA-AMMOS/aerie/pull/1513/commits/d772cf80371c495ddbb7194d36cdb7b5bdcfb61c).